data/mods/Xedra_Evolved/mutations/paraclesians/arvore_eocs.json

```diff
@@ -284,6 +284,30 @@
       }
     ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_ARVORE_SUMMON_FLORAL_BOAT_CHECK",
```

Comment:
I don't see anything turning the boat back into a tree. Turning a tree into a temporary boat, without turning it back afterwards, is effectively tree-murder. (A permanent boat would just be a different shape.) I think the boat should be turned back into a tree at the end, or the boat should be permanent.

Reply:
The boat should not be permanent for balance reasons.

Turning it back into a tree unfortunately isn't easily possible. There's no way to track a vehicle's location using EoCs and vehicles can't be talkers either, so there's nothing to hook an EoC on to, which means there's no way to let you manually turn it back either (you can't reverse it to stand next to the vehicle and place a tree). Especially if the vehicle is out of the reality bubble when it transforms back. 

I can have it turn the tree into a young tree so you can't make the tree disappear, just take a great part of its wood to make the boat. That serves the same purpose of making it ineligible to have the spell cast on it again (unless you use Nurturing the Green and grow the tree again, but that's fine).

Reply:
The "young tree" thing sounds fine to me.